jenkins/automation/kubernetes/setup-cortx-cluster.groovy

```diff
@@ -190,7 +190,7 @@ pipeline {
                 ./cortx-deploy.sh --support-bundle
                 HOST_FILE=$PWD/hosts
                 PRIMARY_NODE=$(head -1 "$HOST_FILE" | awk -F[,] '{print $1}' | cut -d'=' -f2)
-                LOG_FILE=$(ssh -o 'StrictHostKeyChecking=no' $PRIMARY_NODE 'ls -t /root/deploy-scripts/k8_cortx_cloud | grep logs-cortx-cloud | grep .tar | head -1')
+                LOG_FILE=$(ssh -o 'StrictHostKeyChecking=no' $PRIMARY_NODE 'ls -t /root/deploy-scripts/k8_cortx_cloud | grep logs-cortx-cloud | grep .tgz | head -1')
```

Comment:
We can combine two commands in a single as below,
```
scp -q "$PRIMARY_NODE":/root/deploy-scripts/k8_cortx_cloud/logs-cortx-cloud*.tgz $WORKSPACE/artifacts/
```

Reply:
Done

Reply:
Done